docs/notebooks/Kusto-Analysis.ipynb

```diff
@@ -0,0 +1,1642 @@
+{
```

Comment:
<p>You could also use the MSTICPy b64 decoder - it will (should) recursively decode embedded b64 strings as well.</p><p>The output is a bit janky though. - you'd get 2 rows (one for the next B6 block) decoded_string is the column in the output DF.</p><p>Prob not worth worrying about.</p><p> </p><p>Oddly enough I'm working on little function that isn't yet in msticpy to take obfuscated powershell stuff like that and print it out in a more readable form - not anything to concern you, just an interesting coincidence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/msticpy/pull/308/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Changed. Good to do it the MSTICPy way.</p>